microsoft-edge/extensions-chromium/publish/verify-microsoft-edge-program.md

```diff
@@ -0,0 +1,90 @@
+
+
+<!-- ====================================================================== -->
+## Checking your verification status
+
+You can check verification status at Partner Center in **[Account Settings | Legal Info](https://partner.microsoft.com/dashboard/account/v3/organization/legalinfo#developer)**, where you can see:
+
+1. One or more tabs such as **Partner**, **Developer**, or **Reseller**, depending on the programs in which your organization is enrolled.
+1. Your verification status: **Pending**, **Accepted**, or **Rejected**, with a status icon.
+1. The **Legal business profile** progress bar with an information icon you can select to get more information.
+
+
+<!-- ====================================================================== -->
+## Verification status
+
+There are three possible results when you check your verification status:
+
+*   **Accepted**: The information you submitted was verified, and you're notified of your acceptance into the program. No further action is required.
+
+*   **Pending**: The verification process has started but isn't complete. No action is required. You can monitor verification status at Account Settings | Legal Info. Verification usually takes 3-5 business days.
+
+*   **Rejected**: The information you submitted couldn't be verified. The reason and instructions for [how to appeal](#appealing-a-rejected-application) appear in the Account Verification pane.
```

Comment:
The second sentence can be changed to: Visit [Appealing a rejected application ](#appealing-a-rejected-application) section for instructions on how to appeal a rejection.

Reply:
```suggestion
* **Rejected**: The information you submitted couldn't be verified. The reason, and instructions for how to appeal, appear in the **Account Verification** pane. See [Appealing a rejected application](#appealing-a-rejected-application), below.
* ```

Reply:
fixed